test/jasmine/tests/config_test.js

```diff
@@ -551,230 +551,232 @@ describe('config argument', function() {
-                checkElementsSize(canvases, elWidth / 2, elHeight / 2);
+                return Promise.resolve()
+                .then(delay(RESIZE_DELAY))
+                .then(function() {
+                    checkLayoutSize(elWidth / 2, elHeight / 2);
 
-            })
-            .catch(failTest);
-        }
+                    var mainSvgs = document.getElementsByClassName('main-svg');
+                    checkElementsSize(mainSvgs, elWidth / 2, elHeight / 2);
 
-        function fillParent(numRows, numCols, cb) {
-            elWidth = width / numCols, elHeight = height / numRows;
+                    var canvases = document.getElementsByTagName('canvas');
+                    checkElementsSize(canvases, elWidth / 2, elHeight / 2);
 
-            // Fill parent
-            for(var i = 0; i < (numCols * numRows); i++) {
-                var col = document.createElement('div');
-                col.style.height = '100%';
-                col.style.width = '100%';
-                if(typeof(cb) === typeof(Function)) cb.call(col, i);
-                parent.appendChild(col);
+                })
+                .catch(failTest);
             }
-            document.body.appendChild(parent);
-            gd = parent.childNodes[0];
-        }
 
-        it('@flaky should resize when the viewport width/height changes', function(done) {
-            fillParent(1, 1);
-            Plotly.plot(gd, data, {}, {responsive: true})
-            .then(testResponsive)
-            .then(done);
-        });
-
-        it('@flaky should still be responsive if the plot is edited', function(done) {
-            fillParent(1, 1);
-            Plotly.plot(gd, data, {}, {responsive: true})
-            .then(function() {return Plotly.restyle(gd, 'y[0]', data[0].y[0] + 2);})
-            .then(testResponsive)
-            .then(done);
-        });
+            function fillParent(numRows, numCols, cb) {
+                elWidth = width / numCols, elHeight = height / numRows;
+
+                // Fill parent
+                for(var i = 0; i < (numCols * numRows); i++) {
+                    var col = document.createElement('div');
+                    col.style.height = '100%';
+                    col.style.width = '100%';
+                    if(typeof(cb) === typeof(Function)) cb.call(col, i);
+                    parent.appendChild(col);
+                }
+                document.body.appendChild(parent);
+                gd = parent.childNodes[0];
+            }
 
-        it('@flaky should still be responsive if the plot is purged and replotted', function(done) {
-            fillParent(1, 1);
-            Plotly.plot(gd, data, {}, {responsive: true})
-            .then(function() {return Plotly.newPlot(gd, data, {}, {responsive: true});})
-            .then(testResponsive)
-            .then(done);
-        });
+            it('@flaky should resize when the viewport width/height changes', function(done) {
```

Comment:
We should add the `traceType` value inside these `it`s so that if the scatter or the scattergl cases fail, we'll know which one did. For example,

```js
it('@flaky should resize when the viewport width/height changes (case ' + traceType + ')', function(done) {
// ...
```